### PR TITLE
pythonPackages.imantics: init at 0.1.12

### PR DESCRIPTION
**Pull request by Rakesh4G**
###### Motivation for this change
pythonPackages.imantics: init at 0.1.12
Requires : https://github.com/NixOS/nixpkgs/pull/102440
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
You could have put that into the other PR because it is required for the other package.

File: pkgs/development/python-modules/imantics/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    lxml
+    xmljson
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "'opencv-python>=3'," ""
+  '';
+
+  doCheck = False;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/imantics/default.nix:35]: Does it not have tests? If so can you at least do an import check?

File: pkgs/development/python-modules/imantics/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    numpy
+    opencv3
+    sphinx_rtd_theme
+    lxml
+    xmljson
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "'opencv-python>=3'," ""
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/imantics/default.nix:32]:
```suggestion
      --replace "'opencv-python>=3'," "'opencv-python'"
```

File: pkgs/development/python-modules/imantics/default.nix (hunk 1 above)
Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix:35]: Test cases are there, but they are failing. I am not having time to fix them.

File: pkgs/development/python-modules/imantics/default.nix (hunk 2 above)
Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix:32]: This is not intended behaviour. We have seen many application have `opencv-python` in requirements but fail to work when we try on nixos.

File: pkgs/development/python-modules/imantics/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/imantics/default.nix:35]: Can you add a comment about that?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/imantics/default.nix:35]:
```suggestion
  # failing on NixOS
  doCheck = False;
```

File: pkgs/development/python-modules/imantics/default.nix [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  '';
+
+  doCheck = False;
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/imantics/default.nix:37]:
```suggestion
  meta = with lib; {
```

This fixes the eval error.

Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix:37]: This should not be an issue as i have imported lib on top of the package.

File: pkgs/development/python-modules/imantics/default.nix (hunk 1 above)
Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix:35]: ok

File: pkgs/development/python-modules/imantics/default.nix (hunk 3 above)
Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix:37]: oh  sorry, I misread it. correcting

File: pkgs/development/python-modules/imantics/default.nix [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    xmljson
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "'opencv-python>=3'," ""
+  '';
+
+  # failing on NixOS
+  doCheck = False;
```
Comment (jonringer) [inline pkgs/development/python-modules/imantics/default.nix:36]: If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix:36]: ok

Comment (Rakesh4G) [reply pkgs/development/python-modules/imantics/default.nix]: Done

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102442` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python27Packages.imantics</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.imantics</li>
    <li>python38Packages.imantics</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102442` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python27Packages.imantics</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.imantics</li>
    <li>python38Packages.imantics</li>
  </ul>
</details>

**Pull request merged**